packages/server/utils/fromGlobalId.ts

```diff
@@ -0,0 +1,17 @@
+const charSet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_:'
+
+function fromGlobalId(globalId: string) {
```

Comment:
I took approach from  https://github.com/graphql/graphql-relay-js, but wrote my own functions that produces more beautiful global ids

Reply:
Also see #6642

Reply:
Examples:

`Company:123`

graphql-relay: `Q29tcGFueToxMjM=`
own: `8USVGT4Fuvw`

`DomainJoinRequest:99999`

graphql-relay: RG9tYWluSm9pblJlcXVlc3Q6OTk5OTk=
own: `9USGOTcUOTkKW0KYZFCCCCC`

Reply:
own functions produces slightly more shorter ids
graphql-relay function uses base64 and it is probably easier to convert back manually

Reply:
+1 Not sure this is worth introducing our own encoding schema. It's easy enough to look this up in our code. Shorter ids are nice though.

Reply:
updated to use `graphql-relay`